service_container/alias_private.rst

```diff
@@ -43,29 +46,30 @@ public (i.e. private):
             xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
             <services>
-                <service id="foo" class="Example\Foo" public="false" />
+                <service id="AppBundle\Service\Foo" public="false" />
             </services>
         </container>
 
     .. code-block:: php
 
-        use Example\Foo;
+        use AppBundle\Service\Foo;
 
-        $container->register('foo', Foo::class)
+        $container->register(Foo::class)
             ->setPublic(false);
 
-What makes private services special is that, since the container knows that the
-service will never be requested from outside, it can optimize whether and how it
-is instanciated. This increases the container's performance.
+Private services are special because it allows the container to optimize whether
```

Comment:
`because it allows the container [...]` -> `because they allow the container [...]` ?